### PR TITLE
Add Taiwan translation for README.md file

### PR DESCRIPTION
**Pull request by evshary**
Add Taiwan translation for README.md file

### CODE REVIEW

File: README.zh_TW.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+</p>
+
+<p align="center">
+<i>「回想起早期 CentOS 的時光... 我的共同創辦人是 Rocky McGaugh。他已經不在我們身邊了，沒能看到 CentOS 的成功，因此為了向他致敬，我向你們介紹...Rocky Linux」</i><br>
+— Gregory Kurtzer, 創辦人
+</p>
+
+## 關於
+
+為了因應 CentOS 發展方向的改變，Rocky Linux 會是一個基於社群所開發的企業級作業系統，能與 RHEL 發行版 100％ 完全兼容。
```
Comment (toto6038) [inline README.zh_TW.md:12]: "compatible" should be 相容 instead of 兼容

**Review by toto6038 — CHANGES_REQUESTED** [review]

File: README.zh_TW.md (hunk 1 above)
Comment (evshary) [reply README.zh_TW.md]: @toto6038 Thanks for your suggestion. I've updated the documentation.

**Comment by evshary** [comment]
@toto6038 We're happy to see Taiwanese in the Rocky Linux community. If you have any comments, please let us know.
BTW, we've created a [website](https://rockylinux.tw/) to bridge Rocky Linux and Taiwan's local community. Feel free to PR to the repo if you have anything worthy to share with the community.

**Comment by toto6038** [comment]
Glad to see that! It's fast to have a Taiwan domain. Maybe we need a TW telegram group?

**Comment by evshary** [comment]
Well, to be honest, I'm a little sad that #81 is created and merged instead of discussing and modifying this earlier PR.
But that's fine. I'll close this PR.

**Comment by toto6038** [comment]
Ah... sorry for that

**Comment by toto6038** [comment]
Ah... sorry for that. You've done a great job!!! Appreciate for your work :)

**Pull request closed without merging**